website/docs/migrations/11-12.md

```diff
@@ -0,0 +1,158 @@
+
+```php
+if (\class_exists(ManifestCache::class)) {
+	(new ManifestCache())->setAllCache();
+}
+```
+
+### Replace ManifestItem filter with Asset helper
+
+**Migration time: ~2min.**
+
+If you are using the `apply_filters(Manifest::MANIFEST_ITEM, '<key>')` filter in your project, you should replace it with the new `Helpers::getAsset('<key>')` helper.
+
+### Update helpers
+
+**Migration time: ~5min.**
+
+1. Find all instances of `outputCssVariablesGlobal(array $globalManifest = [])` function in your project and remove the global settings variable, eg. `outputCssVariablesGlobal()`.
+
+2. Find all instances of `outputCssVariables(array $attributes, array $manifest, string $unique, array $globalManifest = [], string $customSelector = '')` function in your project and remove the global settings variable, eg. `outputCssVariables(array $attributes, array $manifest, string $unique, string $customSelector = '')`.
+
+### Update all renderPartial methods
+
+**Migration time: ~15min.**
+
+Find all instances of `renderPartial` method in your project and replace it with the new `render` method.
+
+**Before:**
+```php
+echo Components::renderPartial(
+	'component',
+	'utils',
+	'debug-field-details',
+	[
+		'name' => Components::checkAttr('fieldName', $attributes, $manifest),
+	]
+);
+```
+
+**After:**
+```php
+echo Components::render(
+	'debug-field-details',
+	[
+		'name' => Components::checkAttr('fieldName', $attributes, $manifest),
+	],
+	'components',
+	false,
+	'utils/partials'
+);
+```
+
+### Update render method
+
+**Migration time: ~10min.**
+
+Find all instances of `render` method in your project where you are using more than 2 arguments and check the new method signature.
+
+**Before:**
+```php
+return Components::render(
+	'form',
+	Components::props('form', [], $formProps),
+	'',
+	true
+);
+```
+
+**After:**
+```php
+return Components::render(
+	'form',
+	Components::props('form', [], $formProps),
+	'components',
+	true
+);
+```
+
+### Update wrapper and all blocks with innerBlockContent
+
+**Migration time: ~15min.**
+
+1. You should find all instances of `$this->renderWrapperView` method in your project and replace it with the new `echo $renderContent` method.
+2. You should find all instances of `$innerBlockContent` param in your project and replace it with the new `echo $renderContent` method. (Wrapper, Group, Columns, etc.)
+3. Remove `wrapperManualContent` param from all `wrapper` instances.
+You can check the [example](https://github.com/infinum/eightshift-frontend-libs/blob/feature/new-libs/blocks/init/src/Blocks/wrapper/wrapper.php)
+
+**Before:**
+```php
+$this->renderWrapperView(
+	$templatePath,
+	$attributes,
+	$innerBlockContent
+);
+```
+
+**After:**
+```php
+echo $renderContent;
+```
+
+## Optional changes
```

Comment:
I'd bump this to non-optional, as the old versions will be deprecated in the next release.